tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -311,7 +311,7 @@ function kbs_k8s_deploy() {
 		echo "Setting up custom PCCS for TDX"
 		cat <<- EOF > "${COCO_KBS_DIR}/config/kubernetes/custom_pccs/sgx_default_qcnl.conf"
 {
- "pccs_url": "https://$(hostname -i):8081/sgx/certification/v4/",
+ "pccs_url": "https://$(hostname -i | grep -o "^[0-9.]*"):8081/sgx/certification/v4/",
```

Comment:
Nit: This assumes an IPv4 address [*], but, I guess, this is fine if we can guarantee an IPv4 address as the first entry.

---

[*] - and fails on my local system as I get an IPv6 address first.

Reply:
In our CI we guarantee that IPv4 is used.

I need to get in touch with my ISP in order to get IPv6 working here, so I can deal with another set of problems. :-)